tgui/public/tgui.html

```diff
@@ -98,6 +98,11 @@
         }
       }
     }
+    // If we're a Chromium client, just use the fancy method
+    if (typeof cef_to_byond != 'undefined') {
```

Comment:
Replace with

```
if (window.cef_to_byond) {
```

Because otherwise it's a silent access of an undefined variable which can be an exception, but it isn't because IE11 runs in non-strict mode by default, it's not.

So better replace all references with window.cef_to_byond.

Reply:
This isn't necessary for popper at all, do you still want this anyway so we can have good Chromium support?

Reply:
yeah, why not

Reply:
or indeed, you can atomize